Dfe.Academies.External.Web/ViewModels/LoanViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Dfe.Academies.External.Web.ViewModels
+{
+	public class LoanViewModel
+	{
+		public int Id { get; set; }
+		public bool IsDraft { get; set; }
+
+		public string DisplayName { get; set; }
```

Comment:
do you want to do the = null!; at the end of the non-nullable string to remove these build warnings?